packages/framework/composer.json

```diff
@@ -94,6 +94,7 @@
         "ext-pgsql": "Required for dumping the Postgres database in shopsys:database:dump command"
     },
     "conflict": {
-        "symfony/dependency-injection": ">=3.4.15"
+        "symfony/dependency-injection": "3.4.15",
```

Comment:
these settings are also in 
![image](https://user-images.githubusercontent.com/39240194/47085071-cc633500-d215-11e8-8676-b93a9bbc5328.png)

Reply:
Since there is only `symfony/dependency-injection` conflicting in all packages, I changes the conflict to DI only

Reply:
in the end I marked as conflicting 3.4.15 and 3.4.16